deid/cms_deid.sql

```diff
@@ -1519,4 +1528,1060 @@ select /*+ PARALLEL(hospice_demo_codes,12) */
   idt.EXTRACT_DT
 from hospice_demo_codes idt
 join bene_id_mapping bm on bm.bene_id = idt.bene_id;
-commit;
\ No newline at end of file
+commit;
+
+
+insert /*+ APPEND */ into "&&deid_schema".maxdata_ip

```

Comment:
`maxdata_ip` isn't new. Why is there all this new code for it? I don't see any explanation in the c2e4747 commit message.